nativelib/src/main/resources/platform.c

```diff
@@ -0,0 +1,20 @@
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+int scalanative_platform_is_windows() {
```

Comment:
@MasseGuillaume This is incorrect, we do not need to distribute per-platform artefacts for this to work. C files in `nativelib` are distributed as source and compiled on the *target user machine*, this means that can use whatever platform-specific things the platform has as long as those are ifdef guarded to also work on other platforms.